systems/plants/@RigidBodyManipulator/getCMMdA.m

```diff
@@ -1,9 +1,16 @@
-function [A,dA] = getCMMdA(model,kinsol)
+function [A,dA] = getCMMdA(model,kinsol, robotnum, in_terms_of_qdot)
 % returns the centroidal momentum matrix as described in Orin & Goswami 2008
 %
 % h = A*qd, where h(4:6) is the total linear momentum and h(1:3) is the
 % total angular momentum in the centroid frame (world fram translated to COM).
 
+if nargin < 3
+  robotnum = 1;
```

Comment:
we've been updating this.  to be consistent with everything else, robotnum=0 should do the world (really).  and robotnum<0 should do all bodies.